shared/src/main/kotlin/com/egm/stellio/shared/model/GeoQueryUtils.kt

```diff
@@ -0,0 +1,18 @@
+package com.egm.stellio.shared.model
```

Comment:
don't get why what remains from `util/GeoQueryUtils` is now in `model` (actually, its place is in `util`)